source/_integrations/librehardwaremonitor.markdown

```diff
@@ -0,0 +1,86 @@
+---
+title: Libre Hardware Monitor
+description: Instructions on how to integrate Libre Hardware Monitor within Home Assistant.
+ha_category:
+  - System monitor
+ha_release: 2025.4
+ha_iot_class: Local Polling
+ha_domain: librehardwaremonitor
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+ha_quality_scale: bronze
+---
+
+The `librehardwaremonitor` {% term integration %} uses your [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) installation as a source for sensors to display that system information in Home Assistant.
```

Comment:
```suggestion
The **Libre Hardware Monitor** {% term integration %} uses your [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) installation as a source for sensors to display that system information in Home Assistant.
```
we no longer use the code-fenced version to spell the integration name for integrations that have a config flow. you might still see it for those that are added via YAML config file.